src/LineUI/LineUnit.tsx

```diff
@@ -93,19 +93,21 @@ interface ILineUnitProps {
   moveEndCB?: () => void;
   label?: string;
   styling?: string;
+  bothDirectionMark: boolean;
+  hideMoveHandle?: boolean;
```

Comment:
Please change `hideMoveHandle` to `showMoveHandle` 
Also make sure you update the implementation where this variable is used.

Reply:
showMoveHandle is already available in the global option.

Reply:
You can give different name to it but hideMoveHandle is opposite of global prop showMoveHandle which is confusing here.  You can change hideMoveHandle to showLineMoveHandle or something similar.

Reply:
I agree with @fti-bcarpenter  maybe it can be something like overrideShowMoveHandle